web/packages/teleport/src/Account/ManageDevices/AddAuthDeviceWizard/AddAuthDeviceWizard.tsx

```diff
@@ -306,7 +453,25 @@ function SavePasskeyStep({
               onChange={onNameChange}
               readonly={saveAttempt.attempt.status === 'processing'}
             />
-            <ButtonPrimary type="submit">Save the Passkey</ButtonPrimary>
+
```

Comment:
Can't comment on the actual line, but see line 448 above:

`label="Passkey Nickname"`

This input is shown when adding webauthn devices and TOTP authenticators, both of which are not passkeys.

Reply:
Good catch